src/memcached/cache_impl.go

```diff
@@ -83,6 +83,12 @@ func (this *rateLimitMemcacheImpl) DoLimit(
 			continue
 		}
 
+		// Check if the key is unlimited
```

Comment:
It seems odd that we have to hit the cache at all for unlimited limits? Can this be avoided somehow?

Reply:
Moved the logic to ratelimit.go